include/z64player.h

```diff
@@ -585,7 +585,7 @@ typedef struct Player {
     /* 0xAB2 */ Vec3s unk_AB2;
     /* 0xAB8 */ f32 unk_AB8;
     /* 0xABC */ f32 unk_ABC;
-    /* 0xAC0 */ f32 unk_AC0;
+    /* 0xAC0 */ f32 displacementY;
```

Comment:
This would probably be better left for `player` decomp/documentation which is currently ongoing.

Reply:
Agreed, what seems to usually be the case with Player is that something is used for 6 different things in different contexts. I also don't find this a very meaningful name anyway.

Reply:
I'd say kind of a bit of both. Player should almost certainly be avoided, not because it is ongoing, but player is so big it is difficult to know if that is the actual purpose of the variable. It seems reasonable in this context, but it could be used differently in other contexts that might shed more light. Crossover is fine, as long as their is enough context to figure out that is actually the function of the struct member.